releasenotes/notes/no-timeout-for-flares-930505387df0e5c4.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
```

Comment:
Wouldn't this be a fix instead of an enhancement?

Reply:
Maybe?  It depends on whether this was intended behavior or not, I suppose.